packages/dd-trace/src/profiling/exporters/agent.js

```diff
@@ -61,13 +66,22 @@ class AgentExporter {
         options.port = this._url.port
       }
 
+      this._logger.debug(`Submitting agent report to: ${JSON.stringify(options)}`)
```

Comment:
This should be in a callback to avoid the stringify when the logger is disabled.